src/data/dataMethods.ts

```diff
@@ -367,6 +368,27 @@ export function _create<R extends Record<string, Any>>(
   return _dataRequest(client, httpRequest, 'mutate', {mutations: [mutation]}, opts)
 }
 
+const hasDataConfig = (client: SanityClient | ObservableSanityClient) =>
```

Comment:
`type Client = SanityClient | ObservableSanityClient` would make this quite a bit more readable I think